packages/ckeditor5-typing/tests/utils/injectunsafekeystrokeshandling.js

```diff
@@ -129,5 +130,94 @@ describe( 'unsafe keystroke handling utils', () => {
 				return editor.model.change( writer => writer.batch );
 			}
 		} );
+
+		describe( 'handling Shift + Delete on Windows', () => {
+			let view, viewDocument, oldEnvIsWindows;
+
+			beforeEach( () => {
+				view = editor.editing.view;
+				viewDocument = view.document;
+			} );
+
+			describe( 'on Windows', () => {
+				before( () => {
```

Comment:
Is `before` enough? Shouldn't this be `beforeEach`?

Reply:
In my opinion `before` is enough - there is no need to set `env` every time, but just once before the whole test group.